docs/api/hooks.md

```diff
@@ -61,3 +61,23 @@ export const RouterActionsHookExample = () => {
   return <MyComponent push={push} />;
 };
 ```
+
+## useQueryParam
+
+You can use the `useQueryParam` hook to access the query params in current route.
+
+```js
+import { useQueryParam } from 'react-resource-router';
+
+// Current route in address bar — /home/projects?foo=bar
+
+export const MyComponent = () => {
+  const [foo, setFoo] = useQueryParam('foo');
+  // => foo will have the value 'bar'
+
+  setFoo('baz');
```

Comment:
🤯

Reply:
@theKashey is that a "that's awesome" 🤯 or "what the hell are you doing here?" 🤯?

Reply:
Oh yeah, the example is non contextual. I'll update it to invoke the setter on a button event.